### PR TITLE
Feature: Forget Password and OTP verification 

### LINKED ISSUE #13: Feature: forget password 04(21-23)

### PR DESCRIPTION
**Pull request by aimeudoxie**
**What this PR does**
This pull request contains the implementation of forget password and OTP verification screen

**Description**
**Forget password screen**
In this feature a user will be able to select an method that will be used to send the reset password code either by email or by phone.
This is implemented in both light and dark mode
Screenshot 2024-05-06 020815
Screenshot 2024-05-06 020852

**OTP verification screen**
For the user that have chosen to receive the reset code through Phone this feature will help to enter the Reset code sent to his/her phone for it to be verified
This is implemented in both light and dark mode
Screenshot 2024-05-06 021010
Screenshot 2024-05-06 021055

### CODE REVIEW

**Review by Irirwanirira — CHANGES_REQUESTED** [review]
Good job so far, but you still have some conflicts to resolve !!

**Review by Irirwanirira — APPROVED** [review]
Great job

File: README.md [hunk 1]
```diff
@@ -120,12 +130,39 @@ example@gmail.com
...
+<<<<<<< HEAD
+=======
+## License
+
+None At the moment !!
+>>>>>>> bc9ea19 (ft(setup fingerprint) fingerprint screen and its modal component)
+
+=======
+
+>>>>>>> 0cf82c1b70117e205e437bb58220f0ef9ad30984
```
Comment (Nkbtemmy) [inline README.md]: Even here

File: README.md [hunk 2]
```diff
@@ -120,12 +130,39 @@ example@gmail.com
 Feel free to reach out to us:
 [Nkbtemmy](https://github.com/Nkbtemmy)
 [mukunzidd](https://github.com/mukunzidd)
+<<<<<<< HEAD
+=======
 
 ## License
+>>>>>>> 0cf82c1b70117e205e437bb58220f0ef9ad30984
```
Comment (Nkbtemmy) [inline README.md]: This indicates that you still have conflicts

File: components/UI/Header.tsx [hunk 3]
```diff
@@ -50,4 +51,46 @@ export default function Header({ title }: Props) {
       </View>
     </>
   );
+=======
+>>>>>>> a304d8c (ft(auth): adding-forgetPassword-OTPverification)
```
Comment (Nkbtemmy) [inline components/UI/Header.tsx:55]: This also indicates that you still have resolved conflicts not correctly

File: components/UI/icons.tsx [hunk 4]
```diff
@@ -24,6 +24,38 @@ export function LeftArrow({ fillColor }: { fillColor: string }) {
...
+    <Svg width="32" height="33" viewBox="0 0 32 33" fill="none">
+      <Path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M22.5857 4.66663C24.3737 4.66663 26.0937 5.37329 27.359 6.64129C28.6257 7.90663 29.3337 9.61329 29.3337 11.4V21.9333C29.3337 25.6533 26.307 28.6666 22.5857 28.6666H9.41366C5.69233 28.6666 2.66699 25.6533 2.66699 21.9333V11.4C2.66699 7.67996 5.67899 4.66663 9.41366 4.66663H22.5857ZM24.707 13.3866L24.8137 13.28C25.1323 12.8933 25.1323 12.3333 24.799 11.9466C24.6137 11.748 24.359 11.6266 24.0937 11.6C23.8137 11.5853 23.547 11.68 23.3457 11.8666L17.3337 16.6666C16.5603 17.308 15.4523 17.308 14.667 16.6666L8.66699 11.8666C8.25233 11.56 7.67899 11.6 7.33366 11.96C6.97366 12.32 6.93366 12.8933 7.23899 13.2933L7.41366 13.4666L13.4803 18.2C14.227 18.7866 15.1323 19.1066 16.0803 19.1066C17.0257 19.1066 17.947 18.7866 18.6923 18.2L24.707 13.3866Z"
+        fill={fillColor}
+      />
+      </Svg>
+  );}
+>>>>>>> a304d8c (ft(auth): adding-forgetPassword-OTPverification)
```
Comment (Nkbtemmy) [inline components/UI/icons.tsx:58]: Please resolve your conflict correctly

File: constants/Colors.ts [hunk 5]
```diff
@@ -61,9 +61,8 @@ export const Colors = {
     "_50": "#FAFAFA",
   },
 
-  // will require to set this up here as soon as one needs gradients
   // gradients: {
-  //   gradient_blue: { from: { x: 0, y: 0 }, to: { x: 1, y: 1 }, color: [""] },
+  //   gradient_blue: '{ from: { x: 0, y: 0 }, to: { x: 1, y: 1 }, color: [""] }',
   //   gradient_yellow: { from: { x: 0, y: 0 }, to: { x: 1, y: 1 }, color: [""] },
   //   gradient_green: { from: { x: 0, y: 0 }, to: { x: 1, y: 1 }, color: [""] },
   //   gradient_orange: { from: { x: 0, y: 0 }, to: { x: 1, y: 1 }, color: [""] },
```
Comment (Nkbtemmy) [inline constants/Colors.ts:68]: Delete commented codes; in case you aren't using them

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**